TrainingApp/src/components/Card/style.js

```diff
@@ -0,0 +1,13 @@
+import { StyleSheet } from 'react-native';
+
+export default StyleSheet.create({
+  card: {
+    shadowColor: 'black',
```

Comment:
always use HEX codes

Reply:
Removed the Card component as it is not being used anywhere